lib/ruby/lib/alces/stack/forkprocess.rb

```diff
@@ -0,0 +1,106 @@
+#==============================================================================
```

Comment:
This looks like it's only used in the tests too.  Another candidate for relocation?